app/src/main/java/com/framgia/fsalon/data/model/UserRespone.java

```diff
@@ -0,0 +1,34 @@
+package com.framgia.fsalon.data.model;
+
+import com.framgia.fsalon.data.model.Token;
+import com.framgia.fsalon.data.model.User;
```

Comment:
ERROR: Redundant import from the same package - com.framgia.fsalon.data.model.User.